int16 aruco_id

